src/lib/styles/app.css

```
@@ -13,11 +24,139 @@ body {
+/* Re-add hover shading to plan stop button */
+button.message-button:hover {
+    background-color: oklab(0.28 -0.01 -0.03 / 0.15);
+}
+
+/* Fix text color for modal close button */
+.close-btn > .dy-btn-ghost {
+    color: inherit;
+}
+
+/* Make v5 search bar checkboxes more readable */
+@utility dy-checkbox-accent {
```

Stylelint (17.14.0)

[error] Unexpected unknown at-rule "`@utility`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)